cnf/ext/repositories.bnd

```diff
@@ -4,43 +4,47 @@ ossrh:                  https://oss.sonatype.org/content/repositories/snapshots
 
 -plugin.0.Main:\
     aQute.bnd.repository.maven.provider.MavenBndRepository;\
+    	tags  = 'resolve'; \
         name="Maven Central";\
         releaseUrl="${mavencentral}";\
         snapshotUrl="${ossrh}";\
         index="${.}/central.mvn";\
         readOnly=true,\
     aQute.bnd.repository.maven.provider.MavenBndRepository;\
+        tags  = '-'; \
         name="Local";\
         noupdateOnRelease=true,\
     aQute.bnd.repository.maven.provider.MavenBndRepository;\
+    	tags  = '-'; \
         name="Release";\
         releaseUrl="${fileuri;${releaserepo}}";\
         snapshotUrl="${fileuri;${releaserepo}}";\
         noupdateOnRelease=true,\
     aQute.bnd.repository.maven.provider.MavenBndRepository;\
+    	tags  = '-'; \
         name="JFrog";\
         releaseUrl="https://bndtools.jfrog.io/bndtools/libs-release-local/";\
         snapshotUrl="https://bndtools.jfrog.io/bndtools/libs-snapshot-local/";\
         noupdateOnRelease=true
 
 -plugin.1.Eclipse:\
-    aQute.bnd.repository.osgi.OSGiRepository;\
-        name="Eclipse 4.25 (2022-09)";\
-        locations="https://bndtools.jfrog.io/bndtools/bnd-build/eclipse/4.25/index.xml.gz";\
-        poll.time=-1;\
-        cache="${workspace}/cnf/cache/stable/Eclipse-4.25"
+	aQute.bnd.repository.p2.provider.P2Repository; \
+	 	url = https://download.eclipse.org/releases/2024-12/; \
```

Comment:
Instead of larger upgrades, I would recommend to first go from 

2022-09 > 2022-12

using smaller increments (that can then already be merged) usually works better for plugins quite tightly integrated (like Bndtools)